react/src/pages/InfoPage.js

```diff
@@ -137,6 +146,7 @@ const InfoPage = () => {
     const [ showingAuthToken, setShowAuthToken ] = useState(false);
     const [ showingAccounts, setShowingAccounts ] = useState(false);
     const [ reloadingApp, setReloadingApp ] = useState(false);
+    const accessKey = useFetch("/portal_access_key");
```

Comment:
Make corresponding update to this API internally so it does not acquire the secret.

Reply:
Done, no longer returns any part of the of the secret, just ******.